### PR TITLE
Fix go vet issues

### PR DESCRIPTION
**Pull request by nebril**
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/5128)

### CODE REVIEW

**Comment by nebril** [comment]
test-me-please

File: cilium/cmd/endpoint_config.go [hunk 1]
```diff
@@ -84,7 +84,7 @@ func configEndpoint(cmd *cobra.Command, args []string) {
 		if err != nil {
 			Fatalf("Cannot parse option %s: %s", opts[k], err)
 		}
-		modifiedOptsCfg.Options[name] = fmt.Sprintf("%d", value)
+		modifiedOptsCfg.Options[name] = fmt.Sprintf("%t", value)
```
Comment (joestringer) [inline cilium/cmd/endpoint_config.go]: `ParseOption()` now returns an integer, not a bool - I think that your local cache is out of date, perhaps?

Comment (joestringer) [reply cilium/cmd/endpoint_config.go]: Confirmed by CI failure:
```
12:37:43 cilium/cmd/endpoint_config.go:87: Sprintf format %t has arg value of wrong type int
```

**Comment by nebril** [comment]
Seems like problem on my side was caused by our make recipe using "go tool vet" instead of "go vet", which is suggested as a proper way to call vet since 1.10.

**Comment by joestringer** [comment]
@nebril In that case perhaps we should update this make target to use the right way:

https://github.com/cilium/cilium/blob/9a2623c4143ff5da90306227abc84740dd72583d/Makefile#L213

**Comment by nebril** [comment]
@joestringer just did that!

**Comment by nebril** [comment]
test-me-please

**Review by joestringer — APPROVED** [review]

**Review by aanm — APPROVED** [review]

**Comment by ianvernon** [comment]
Build failed in Jenkins Precheck: 
https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-Validated/5525/execution/node/41

Trying again to see if this is transient, or related to changes in this PR.

**Comment by ianvernon** [comment]
test-me-please

**Comment by ianvernon** [comment]
@nebril these changes broke the Jenkins precheck stage: https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-Validated/5564/execution/node/41/log/

**Comment by nebril** [comment]
Seems like go version might be too low on Jenkins to run `go vet` properly with this setup. Will investigate.

**Comment by nebril** [comment]
test-me-please

**Pull request closed without merging**